docs/graphcache/schema-awareness.md

```diff
@@ -107,9 +107,9 @@ fetch('http://localhost:3000/graphql', {
   });
 ```
 
-The `getIntrospectionQuery` doesn't only accept `IntrospectionQuery` JSON data as inputs, but also
+The `minifyIntrospectionQuery` doesn't only accept `IntrospectionQuery` JSON data as inputs, but also
```

Comment:
I think this was meant to say `getIntrospectionSchema` as it allows the conversion of multiple inputs to the JSON format that `minifyIntrospectionSchema` expects 🤔